### ISSUE TITLE
Multiple glossary terms for same definition appear on the same line

### ISSUE THREAD

**Issue by cblegare** [pradyunsg/furo#35]
**Describe the bug**

Multiple glossary terms for same definition appear on the same line.

**To Reproduce**

Use the following RST or equivalent markdown

```rst
.. glossary::

   term 1
   term 2
      Definition of both terms.
```

**Expected behavior**

both terms should appear on different lines

**Screenshots**

![image](https://user-images.githubusercontent.com/2456133/99846201-ea619900-2b43-11eb-82a9-ba2c92a43a09.png)

**Desktop (please complete the following information):**

- OS: Kubuntu 20.04
- Browser: any

**Comment by pradyunsg** [comment]
Thanks for the report! If you're familiar with CSS/HTML, could you look into how Sphinx RTD theme handles these?

**Comment by cblegare** [comment]
hello @pradyunsg 

Thank you for your feedback!  I really am not that much familiar with CSS/HTML, but I do know that by default multiple `<dt>` element appear on separate line.

```html
<dl>
  <dt>term 1</dt>
  <dt>term 2</dt>
  <dd>Definition of both terms.</dd>
</dl>
```

Renders as

<dl>
  <dt>term 1</dt>
  <dt>term 2</dt>
  <dd>Definition of both terms.</dd>
</dl>

I think it might be possible to fix this by removing `display: inline-block` here: https://github.com/pradyunsg/furo/blob/8586d36f0fde62752fc5c8cfaffa278f20a7f20c/src/furo/assets/styles/content/_lists.sass#L38

but I really am out of my comfort zone here.

**Comment by pradyunsg** [comment]
Ah fun. I'll need to look into where I messed up. Footnotes are also `dl`s in Sphinx, so maybe that in line block is only necessary for them?

🙃

**Comment by pradyunsg** [comment]
Fixed in c3165d35c1ba1051d7f64749a8603708d13445ce. :)

**Comment by pradyunsg** [comment]
Released the fix as part of beta 19. Let me know if that's not good-enough. :)
